src/opm/output/eclipse/FileService/EclOutput.cpp

```diff
@@ -33,10 +33,46 @@
 #include <stdexcept>
 #include <typeinfo>
 
-EclOutput::EclOutput(const std::string& inputFile, bool formatted) :
-  isFormatted(formatted)
+#include <boost/filesystem.hpp>
+
+EclOutput::EclOutput(const std::string&            filename,
+                     const bool                    formatted,
+                     const std::ios_base::openmode mode,
+                     const std::streampos          writePos)
```

Comment:
I think it is slightly misleading to pass a general `streampos` argument in the constructor, that will in general end up in the middle of a keyword and render the file useless; I think the seek should be implemented with context - so that it is guaranteed to be at the start of a keyword.